frontend/src/pages/Graphing/GraphingEditor.tsx

```diff
@@ -628,6 +629,16 @@ export const GraphingEditor = () => {
 		return baseArray.concat(numericKeys).slice(0, 8)
 	}, [numericKeys, keysQuery])
 
+	tempMetricViewTitle.current = useMemo(()=>{
+		let newValue = ''
+		const stringifiedFunctionType = functionType?.toString() ?? ''; 
+		newValue =  (metricViewTitle || stringifiedFunctionType || '');
+		if(newValue === stringifiedFunctionType && stringifiedFunctionType) {
+			newValue+= metric ? `(${metric})` : '';
```

Comment:
could we make the title `Count of Logs` - adding the source name (ie. Logs, Traces, Sessions, etc.) to the string here?

for example (manually manipulating the HTML):
![Screenshot from 2024-05-17 10-31-05](https://github.com/highlight/highlight/assets/1351531/2c622753-c57f-4d29-affd-53ffe3e83854)
![Screenshot from 2024-05-17 10-31-26](https://github.com/highlight/highlight/assets/1351531/5a28829d-ed18-48bd-b34e-fe52d1592783)

Reply:
@Vadman97 Will update the pr.